### PR TITLE
Add Nic.ir.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by pipboy96** [comment]
Website has some mixed content issues, so default_off is fine for now I guess.

File: src/chrome/content/rules/Nic.ir.xml [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-	m: certificate mismatch
-	r: connection refused
-	s: self-signed certificate
-	t: timeout on https
+	Expired:
+		- tools
+		- whois
 -->
-<ruleset name="Nic.ir">
+<ruleset name="Nic.ir" default_off="cert-chain">
```
Comment (J0WI) [inline src/chrome/content/rules/Nic.ir.xml]: @pipboy96 this does only affect `rrmt.nic.ir`.

Comment (pipboy96) [reply src/chrome/content/rules/Nic.ir.xml]: There's other breakage, not sure if splitting this into two rulesets and making one default_off and whitelisting the other is worth it.

Comment (pipboy96) [reply src/chrome/content/rules/Nic.ir.xml]: Feel free to make a follow-up PR if you want to. I will merge it but I don't want to make one myself.

Comment (J0WI) [reply src/chrome/content/rules/Nic.ir.xml]: #17634

**Pull request merged**